app/lib/features/news/widgets/news_side_bar.dart

```diff
@@ -83,7 +83,7 @@ class NewsSideBar extends ConsumerWidget {
         const SizedBox(height: 10),
         InkWell(
           onTap: () {
-            context.pushNamed(
+            context.goNamed(
```

Comment:
We need to fix the router overall, this should really be a push as the user probably wants to navigate back easily, but we can't offer that here. This fix is fine, this is just a remark how bad our state of router is (:( .